desktop-exporter/app/components/detail-view/detail-view.tsx

```diff
@@ -16,7 +16,18 @@ export function DetailView(props: DetailViewProps) {
   let numEvents = span.events.length;
   let numLinks = span.links.length;
 
-  console.log(span);
+  let eventsTab = numEvents ? (
+    <Tab>Events({numEvents})</Tab>
+  ) : (
+    <Tab isDisabled>Events(0)</Tab>
+  );
```

Comment:
This is fine but I wanted to show you another way you could write this:

```jsx
<Tab isDisabled={numEvents === 0}>
  Events({numEvents})
</Tab>
```

Reply:
Yes this is good I like this one.